addons/binding/org.openhab.binding.windcentrale/ESH-INF/thing/millThing.xml

```diff
@@ -36,7 +36,7 @@
 					<option value="121">De Trouwe Wachter</option>
 					<option value="131">De Blauwe Reiger</option>
 					<option value="141">De Vier Winden</option>
-					<option value="201">De Boerenzwaluw</option>
+					<option value="191">De Boerenzwaluw</option>
```

Comment:
I don't get any data when I use the new ID whereas the old one does return data.

You can easily check it in your browser by opening

Existing ID: https://zep-api.windcentrale.nl/production/201/live?ignoreLoadingBar=true

New ID: https://zep-api.windcentrale.nl/production/191/live?ignoreLoadingBar=true

Reply:
I think you are right. I noticed the discrepancy while reviewing
https://zep-api.windcentrale.nl/app/config?p=a
which strangely has 191 as the millId instead of 201